overlord/ifacestate/handlers.go

```diff
@@ -303,3 +306,24 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 	setConns(st, conns)
 	return nil
 }
+
+func (m *InterfaceManager) doDiscardNamespace(task *state.Task, _ *tomb.Tomb) error {
```

Comment:
Why is this in the interface manager?  Isn't that a foundational property of every snap?